### PR TITLE
[lmu2png] Draw event improvements

### LINKED ISSUE #82: [lmu2png] Missing big charset support
The event drawing code is hard-coded to use regular charset dimensions, rendering it incompatible with EasyRPG's big charset feature.

### PR DESCRIPTION
**Pull request by lumiscosity**
Closes #82.
Adds two new flags:
--ic is ignore condition; instead of checking for the first page that has no conditions, it always draws the first page of the event (so like how it appears in the editor)
--sm is simulate movement; for standing animation type events, it sets the center frame instead of the one selected in the editor (so like how they appear in-game) 
Also adds support for big charsets, short hands for the layer commands and cleans up the help command somewhat.

### CODE REVIEW

**Comment by carstene1ns** [comment]
jenkins: ok to test

File: lmu2png/src/main.cpp [hunk 1]
```diff
@@ -118,22 +130,25 @@ void DrawTiles(SDL_Surface* output_img, stChipset * gen, uint8_t * csflag, std::
...
+			for (int i = 0; i < (int)ev.pages.size(); ++i) {
+				const auto& flg = ev.pages[i].condition.flags;
+				if (flg.switch_a || flg.switch_b || flg.variable || flg.item || flg.actor || flg.timer || flg.timer2)
+					continue;
+				evp = &ev.pages[i];
+			}
 		}
 		if (!evp)
-			continue;
+				continue;
```
Comment (carstene1ns) [inline lmu2png/src/main.cpp:148]: Please check for unnecessary whitespace changes.

File: lmu2png/src/main.cpp [hunk 2]
```diff
@@ -145,15 +160,20 @@ void DrawEvents(SDL_Surface* output_img, stChipset * gen, std::unique_ptr<lcf::r
 			}
 
 			SDL_Surface* charset_img(LoadImage(charset.c_str(), true));
-			SDL_Rect src_rect {(evp->character_index % 4) * 72 + evp->character_pattern * 24,
-				 (evp->character_index / 4) * 128 + evp->character_direction * 32, 24, 32};
+			SDL_Rect src_rect;
+			if (!simulate_movement || (simulate_movement && evp->animation_type))
```
Comment (carstene1ns) [inline lmu2png/src/main.cpp:164]: This condition makes no sense to me, can you explain?

File: lmu2png/src/main.cpp (hunk 1 above)
Comment (lumiscosity) [reply lmu2png/src/main.cpp:148]: my bad

File: lmu2png/src/main.cpp (hunk 2 above)
Comment (lumiscosity) [reply lmu2png/src/main.cpp:164]: simulate movement replicates a common trick where events set to the "standing animation" anim type are given a charset with editor-only icons on the sides and an empty row in the middle. when the game runs, it updates the event to show the middle row instead of the side rows. this condition ensures that the simulate movement option is only applied to standing animation anim type events

**Review by fdelapena — APPROVED** [review]

**Comment by lumiscosity** [comment]
should be fixed now! i also applied --sm to the other movement types that exhibited the same properties
